OpenJPEG has been updated in pillow-wheels (#5152)
python-pillow/Pillow: docs/releasenotes/8.1.0.rst

Security
========

> OpenJPEG in the macOS and Linux wheels has been updated from 2.3.1 to 2.4.0, resolving
> security problems present in the earlier version of that dependency.

Reviewer: ```suggestion
OpenJPEG in the macOS and Linux wheels has been updated from 2.3.1 to 2.4.0, including security fixes.
```
